src/components/PokeCard.js

```diff
@@ -1,14 +1,38 @@
-import React from "react";
+import React, { useState } from "react";
+import { Modal as CustomModal } from "./Modal";
+import Modal from "react-modal";
 
 const PokeCard = ({ name, dexNo, typeTags, sprite }) => {
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+
+  Modal.setAppElement("#root");
+
   return (
     <div className="card">
       <ul>
-        <img src={sprite} alt={`${name} sprite`} className="sprite" />
+        <li>
+          <img src={sprite} alt={`${name} sprite`} className="sprite" />
+        </li>
         <li>{name}</li>
         <li>{dexNo}</li>
         <li>{typeTags}</li>
+        <li>
+          <button onClick={() => setModalIsOpen(true)}>Open</button>
+        </li>
       </ul>
+      <Modal
+        className="modalWindow"
+        isOpen={modalIsOpen}
+        onRequestClose={() => {
+          return setModalIsOpen(false);
+        }}>
+        <CustomModal
+          setIsOpen={setModalIsOpen}
```

Comment:
Does this work without passing a parameter to the setter?

Reply:
No it doesn't...  This is a custom React-Modal package we installed.

Reply:
And the React-Modal must have the <Modal> tag while our Modal was renamed to <CustomModal>